### PR TITLE
Create and Add Favicon Issue!

### PR DESCRIPTION
**Pull request by ghost**
See if it works!   @smsunarto @tommyip

### CODE REVIEW

**Comment by tommyip** [comment]
Hey thanks for working on this! You only have to link it in the _includes/head.html file once, no need to put them individually in each file. Also can you put all of this in one commit since they serve a single purpose.

And change the commit title to something like "Add favicon.ico"

File: _layouts/post.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 ---
-
+<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico?">
```
Comment (Arthelon) [inline _layouts/post.html]: You seem to have misspelled favicon.ico ("favicon.ico?")

**Review by Arthelon — CHANGES_REQUESTED** [review]
Minor typo on top of Tommy's previous comments.

File: _layouts/post.html (hunk 1 above)
Comment (Arthelon) [reply _layouts/post.html]: Am aware it doesn't stop file from resolving but it's misleading. Also make sure you keep your branch up to date with master!

**Comment by tommyip** [comment]
Hmm why is this closed?

**Pull request closed without merging**